instagram/src/Components/Authentication/Authenticate.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import Login from '../Login/Login'
+
+const Authenticate = App =>
+	class extends Component {
+		state = {
+			loggedIn: false,
+			username: ''
+		}
+
+		componentDidMount() {
+			if (
+				localStorage.getItem('username') &&
+				localStorage.getItem('password')
+			) {
+				this.setState({
+					loggedIn: true,
+					username: localStorage.getItem('username')
+				})
+			} else {
+				this.setState({
+					loggedIn: false
+				})
+			}
+		}
+		render() {
+			if (this.state.loggedIn === true) {
+				return <App />
+			} else if (this.state.loggedIn === false) {
```

Comment:
Another syntax to this statement is removing the else if, and just adding `return <Login /> ` immediately after the end brace of the if statement.